File: app/src/main/java/xyz/farshad/vocab/model/Level.java
package xyz.farshad.vocab.model;

import com.orm.SugarRecord;

/**
 * Created by farshad on 7/14/16.
 */
public class Level extends SugarRecord {
    String name;
    int courseId;

    public Level() {
    }

    public Level(String name, int courseId) {
        this.name = name;
        this.courseId = courseId;
    }

    public String getName() {
        return name;
    }

    public int getCourseId() {
        return courseId;
    }
}

File: app/src/main/java/xyz/farshad/vocab/model/Level.kt
package xyz.farshad.vocab.model

import com.orm.SugarRecord

/**
 * Created by farshad on 7/14/16.
 */
class Level : SugarRecord {
    var name: String
        internal set
    var courseId: Int = 0
        internal set

    constructor() {}

    constructor(name: String, courseId: Int) {
        this.name = name
        this.courseId = courseId
    }
}
